### PR TITLE
Fix some changelog text

### PR DESCRIPTION
**Pull request by matt-auckland**
I believe this wording is more accurate as I can 'resolve' OpenAlias addresses via RPC commands when they don't have DNSSEC but can't send payments to them when they don't have DNSSEC. Please correct me if I am wrong.

### CODE REVIEW

File: doc/release-notes/release-notes-4.3.0.md [hunk 1]
```diff
@@ -52,15 +52,15 @@ The Wallet GUI is now fully translated to Spanish and Italian.
...
-- Added support for sending to OpenAlias addresses in the GUI, when parsing URIs and the RPC commands validateaddress and sendtoaddress
-- New argument `-requirednssec` to set whether DNSSEC validation is required to resolve openalias addresses (true by default)
+- Added support for sending to OpenAlias addresses in the GUI, when parsing URIs and the RPC commands `validateaddress` and `sendtoaddress`
+- New argument `-requirednssec` to set whether DNSSEC validation is required to resolve an OpenAlias addresses for transactions (true by default)
 - New argument `-mininputvalue` to set the minimum value for an output to be considered as a possible coinstake input
  -New argument `-banversion` to ban nodes depending on their broadcasted version
 - Added support to ban nodes with determined wallet versions using the config parameter `banversion`
 - Blocks created with the rpc command `generate` now include a correct timestamp for the coinbase transaction
-- Using the regtest will create a new genesis block on runtime
+- Using the regtest will create a new genesis block on runtime if there is no existing chaindata
```
Comment (aguycalled) [inline doc/release-notes/release-notes-4.3.0.md]: the new genesis block will be always created

**Comment by matt-auckland** [comment]
Didn't occur in my case, when I launched with data in the folder it
complained and exited with an error

**Comment by matt-auckland** [comment]
I can replicate this for you tomorrow if you wish

File: doc/release-notes/release-notes-4.3.0.md [hunk 2]
```diff
@@ -52,15 +52,15 @@ The Wallet GUI is now fully translated to Spanish and Italian.
...
 - A new testnet has been initiated
 - The development-focused networks regtest and devnet won't require peers for blocks generation
 - Update copyright notice
 - Updates automake to enable dev compilation on New OS's like Ubuntu 18.04
```
Comment (matt-auckland) [inline doc/release-notes/release-notes-4.3.0.md]:
```suggestion
- Updates automake to enable dev compilation on new OS's like Ubuntu 18.04
```

Typo alert!

**Pull request closed without merging**